0.download_data/scripts/download_plates.py

```diff
@@ -71,6 +71,14 @@
         # save extracted zip file from figshare download to a folder in the `0.download_data` directory
         "output_dir": pathlib.Path("./Plate_4_zip"),
     },
+    # need to be extracted in a second step
```

Comment:
I'm not sure what this comment means... Will this notebook perform this extraction? Or is the user expecting to perform some manual step? 

Also, the notebook is missing a print statement for plate 5

Reply:
I have clarified this, it is a mis-copy and paste. It should say that the extraction is performed as a second step. This is fixed.